server/src/core/server/locales/fi-FI/email.ftl

```diff
@@ -84,8 +84,8 @@ email-template-accountNotificationDeleteRequestCompleted =
 # Notification
 
 email-footer-notification =
-  <br /><br />Ystävällisin terveisin<br />$organizationName<br /><br />
-  <i>Etkö enää halua tällaisia viestejä? <a data-l10n-name="unsubscribeLink">Paina tästä</a></i>
+  <br /><br />Ystävällisin terveisin<br />{ $organizationName }<br /><br />Etkö enää halua tällaisia viestejä? <a data-l10n-name="unsubscribeLink">Paina tästä</a>
```

Comment:
I see that the `en-US` translation for this also includes an organization link like so: `<a data-l10n-name="organizationLink">{ $organizationName }</a>`. Could you add it in here as well while you're updating?